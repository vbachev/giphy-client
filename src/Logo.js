import React from 'react'

const Logo = () => (
  <div className='logo-container'>
    <img className='logo-mark' src='giphy-logo.png' alt='Giphy Client'/>
    <h1 className='logo-type'>
      GIPHY
    </h1>
  </div>
)

export default Logo