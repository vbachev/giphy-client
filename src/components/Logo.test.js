import React from 'react'
import renderer from 'react-test-renderer'
import Logo from './Logo'

test('it renders a logo', () => {
  const component = renderer.create(<Logo />)
  expect(component.toJSON()).toMatchSnapshot()
})
