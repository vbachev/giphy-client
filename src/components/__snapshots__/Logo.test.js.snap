// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders a logo 1`] = `
<div
  className="logo-container"
>
  <img
    alt="Giphy Client"
    className="logo-mark"
    src="giphy-logo.png"
  />
  <h1
    className="logo-type"
  >
    GIPHY
  </h1>
</div>
`;
